packages/website/src/pages/layout/Chart.tsx

```diff
@@ -1,224 +1,297 @@
+                    <XYChart series={data} height={height} width={width}>
+                        <XYAxis x={{innerPadding: 30}} y={{innerPadding: 30}}>
+                            <XGrid padding={30} />
+                            <YGrid padding={30} />
+                            <LineSeries />
+                            <InfoLine value={3} label="Three" padding={30} />
+                            <InfoLine value={2} label="Two" padding={30} isVertical />
+                        </XYAxis>
+                    </XYChart>
+                )}
+            />
+        </div>
     );
-};
+
+    const data = [
+        {
+            label: 'First',
+            data: [
+                {x: -1, y: 3},
+                {x: 0, y: 3},
+                {x: 1, y: 3},
+                {x: 2, y: 6},
+                {x: 3, y: 2},
+                {x: 4, y: 12},
+                {x: 5, y: 8},
+            ],
+        },
+        {
+            label: 'Second',
+            data: [
+                {x: -1, y: 1},
+                {x: 0, y: 5},
+                {x: 1, y: 4},
+                {x: 2, y: 0},
+                {x: 3, y: 6},
+                {x: 4, y: 7},
+                {x: 5, y: 4},
+            ],
+        },
+        {
+            label: 'Third',
+            data: [
+                {x: -1, y: 4},
+                {x: 0, y: 7},
+                {x: 1, y: 1},
+                {x: 2, y: 1},
+                {x: 3, y: 1},
+                {x: 4, y: 2},
+                {x: 5, y: 7},
+            ],
+        },
+    ];
+`;
+
+const dateChart = `
+    import * as React from 'react';
+    import moment=require('./node_modules/moment/ts3.1-typings/moment');
+    import {BarSeries, ChartTooltip, ChartContainer, XYAxis, XYChart} from '@coveord/plasma-react';
+
+    export default () => (
+        <div style={{height: '500px'}}>
+            <ChartContainer
+                renderChart={(width, height) => (
+                    <XYChart
+                        series={dateData}
+                        height={height}
+                        width={width}
+                        xFormat={(value: number) => moment.unix(value).format('YYYY-MM-DD')}
+                    >
+                        <XYAxis x={{tickTextSize: 120}} y={{show: false}}>
+                            <BarSeries />
+                            <ChartTooltip sort />
+                        </XYAxis>
+                    </XYChart>
+                )}
+            />
+        </div>
+    );
+
+    const dateData = [
+        {
+            label: 'First',
+            data: Array.from(Array(25).keys()).map((i: number) => ({
+                x: moment().startOf('day').subtract(i, 'day').unix(),
+                y: i + 1,
+            })),
+        },
+    ];
+`;
+
+export const ChartExamples = () => (
+    <PageLayout
+        id="ChartContainer"
+        componentSourcePath="/chart/ChartContainer.tsx"
+        title="Charts"
+        layout="vertical"
```

Comment:
I think it looks better in horizontal, it takes less space (same for the other examples).

![image](https://user-images.githubusercontent.com/35579930/158674367-dfb5514f-73c3-40ee-acf3-3238500af41d.png)